src/test/java/core/basesyntax/operations/ObtainingHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitOperationDto;
+import core.basesyntax.model.OperationType;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ObtainingHandlerTest {
+    private static OperationHandler operationHandler;
+    private FruitOperationDto fruitOperationDto;
+    private Fruit fruit;
+    private int expected;
+    private int actual;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandler = new ObtainingHandler();
+    }
+
+    @Before
+    public void setUp() {
+        fruit = new Fruit("banana");
```

Comment:
could we make our fruit static and initialize it in @BeforeClass method? I think there is no need to create the same fruit before each test (Fix in all classes)